### DISCUSSION TITLE
Bruno changes last digits to zeros when receives long numbers in get response

### DISCUSSION THREAD

**Discussion by IGGO0615** [usebruno/bruno#2830] [Q&A] [answered]
Hi, is anyway to avoid this in response?
Looks like it is using  IEEE 754 spec. I am trying GET API to get SIM number, but Bruno calculate floating-point numbers and returns:

![image](https://github.com/user-attachments/assets/22297791-e81b-47f1-8818-7a90cf158afa)

Can confirm that the same API working fine from other tools and browser

**Comment by helloanoop** [comment, marked as answer]
This has been fixed in `v1.29.0`

Note: If your response is Json, and it contains BigInt values, they may become corrupted during parsing because Bruno automatically processes the response data using JSON.parse(), which doesn’t natively support BigInt. This can result in a loss of precision for these values.

To prevent automatic parsing and work directly with the raw response data, add the below expression to the pre-request script.
```javascript
req.disableParsingResponseJson();
```

Please see doc: https://docs.usebruno.com/advanced-guides/working-with-bigint
